src/applications/managing-app-configuration.md

```diff
@@ -52,6 +52,19 @@ the application to an external PostgreSQL database.
 Detachments can be performed with `deis config:unset`.
 
 
+# Slugbuilder Cache
+By default, apps using the slugbuilder will have caching turned on. This means that deis will
+persist all data being written to `/tmp/cache` inside the buildpack will be persisted between
```

Comment:
I'd say `CACHE_DIR`, which is the [phrasing heroku uses](https://devcenter.heroku.com/articles/buildpack-api#bin-compile). The location `/tmp/cache` isn't guaranteed (it could move in the future) and we don't want users writing to it directly.